docs/Troubleshooting.RunningTests.md

```diff
@@ -25,6 +31,40 @@ detox test --loglevel trace
 
 See [here](https://github.com/wix/detox/blob/master/docs/Troubleshooting.Synchronization.md#identifying-which-synchronization-mechanism-causes-us-to-wait-too-much).
```

Comment:
By looking at this, I recalled that we already have it enabled by default... this section needs an extra review someday.

Reply:
We need a documentation revamp. This is just one symptom.